cmd/kafkalock/kafkalock.go

```diff
@@ -89,7 +91,16 @@ func main() {
 
 	lock, err := kafkalock.NewKafkaLocker(config)
 	if err != nil {
-		logger.Fatal("failed to create kafkaLock: ", err)
+		logger.Fatal("failed to create kafkaLock:", err)
+	}
+
+	if *flap {
+		for {
+			logger.Info("flap")
```

Comment:
mb more verbose?

Reply:
it's just a tool for development, so it doesn't really matters ... but sure, changed